api4/webhook_test.go

```diff
@@ -401,6 +414,20 @@ func TestGetOutgoingWebhooks(t *testing.T) {
 		t.Fatal("missing hook")
 	}
 
+	// Test Team User Case: Permission to MANAGE_OUTGOING_WEBHOOKS but not to MANAGE_OTHERS_OUTGOING_WEBHOOKS
```

Comment:
Parallel feedback to the incoming case above.

Reply:
✅Extracted to its own unit test as well.